tests/main/snap-quota-journal/task.yaml

```diff
@@ -0,0 +1,96 @@
+summary: Functional test for journal quota-related snap commands.
+
+details: |
+  Functional test for snap journal quota group commands ensuring that they are 
+  effective in practice.
+
+# these systems do not support journal quota groups due to their old systemd versions.
+# requires systemd v245+
+systems:
+  - -ubuntu-14.04-*
+  - -centos-7-*
+  - -centos-8-*
+  - -debian-10-64
+  - -amazon-linux-2-*
+  - -ubuntu-16.04-*
+  - -ubuntu-core-16-*
+  - -ubuntu-18.04-*
+  - -ubuntu-core-18-*
+
+prepare: |
+  snap install test-snapd-stressd --edge
+  tests.cleanup defer snap remove --purge test-snapd-stressd
+  snap set system experimental.quota-groups=true
+  tests.cleanup defer snap unset system experimental.quota-groups
+
+restore: |
+  echo "Stopping the service"
+  snap stop test-snapd-stressd.logger
+
+execute: |
+  echo "Starting service and verifying that log messages are sent to default journal namespace"
+  snap start test-snapd-stressd.logger
+
+  # The logger service prints 5 messages every 5 seconds, so lets wait 5 seconds to ensure
+  # that messages appear inside journalctl
+  sleep 5
+
+  # Now ask for logs and lets see if we can see messages from 'LogDaemon'
+  # and at the same time lets get the highest message number to compare with later.
+  # The format of the log messages are 'LogDaemon: message number: X'
+  journalctl | MATCH "LogDaemon"
```

Comment:
you could use here the journal-state tool, that tool will just print the journal logs for the current test.

Reply:
Fixed!